addons/crm/tests/test_crm_lead.py

```diff
@@ -560,7 +560,7 @@ def test_crm_lead_unlink_calendar_event(self):
                 'stop': '2022-07-13 10:00:00',
             }
         ])
-        self.assertEqual(lead.calendar_event_count, 1)
+        self.assertEqual(len(lead.calendar_event_ids), 1)
```

Comment:
Could probably add some checks on the new fields while being there, that way it is covered.